website/src/components/content/WhitePaperPage.tsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react'
+import { RequestDemoAction } from '../../css/components/actions/RequestDemoAction'
+import { COLORS } from '../Jumbotron'
+import { ContentPage } from './ContentPage'
+
+interface Quote {
+    quote: string
+    author: string
+    image?: string
+}
+
+interface Props {
+    title: string
+    quote?: Quote
+    pdf?: string
+    className?: string
+    titleClassName?: string
+    children?: React.ReactNode
+    bgImage?: string
+}
+
+export const WhitePaperJumbotron: React.FunctionComponent<{
+    className?: string
+    color?: keyof typeof COLORS
+    children?: React.ReactNode
+}> = ({ className = '', color = 'dark', children }) => (
+    <div className={`jumbotron rounded-0 ${COLORS[color]} ${className}`}>
+        <div className="container text-center pt-5 pb-3">
+            <span className="white-papers__label d-block mt-1">white paper</span>
+            {children}
+        </div>
+    </div>
+)
+
+export const MediaQuote: React.FunctionComponent<{
```

Comment:
This doesn't seem to be used.

Reply:
Ah true! Deleted it first, then re-added, then decided against again but forgot to remove code 😆.